frontend/js/map.js

```diff
@@ -1,5 +1,3 @@
-mapboxgl.accessToken = 'pk.eyJ1IjoiYWpjaHUyOCIsImEiOiJja3o2M3MzMWswd200MnZwNGdieTNlaHRjIn0.BePZiTybP8rLoo6yQKon_w'
```

Comment:
if u remove this accessToken from Allie + Ying's code then their maps don't work. we need to put this back into their file to avoid breaking it.

Reply:
That was a mistake, not sure when that happened, but prob explains why their map wasn't loading for me.